### DISCUSSION TITLE
Search only works for page titles

### DISCUSSION THREAD

**Discussion by masterbuck** [docmost/docmost#576] [Help / Questions] [answered]
Hi,

I just installed Docmost v0.6.2 via Docker compose and I can't seem to search for anything inside page contents, only in page titles.

This doesn't seem to match with the description made by others of full-text search.

Is there something to activate to make it full text search?

**Comment by Philipinho** [comment]
The search is powered by Postgres full-text search and we index page content too.
What language is your content written in?

**Reply by statistic1900** [reply]
I am experiencing a similar problem as TS, I am in Serbia and I am writing in Cyrillic.

**Reply by masterbuck** [reply]
My content is written in French.

Just tested with a new page called “Test page” and the content “Wrong answer”

Searching for “Test” brings up the page, but searching for “Wrong” brings up nothing.

**Reply by Philipinho** [reply]
@masterbuck I tried the exact words, even with their French translations and the search results were positive.

What version of Postgres are you on?

**Reply by masterbuck** [reply]
I'm on postgres 16-alpine, as recommended on the official installation doc for docker compose: https://docmost.com/docs/installation/

**Reply by mstake01** [reply]
Hi! I confirm the existence of this problem. I am from Ukraine and write notes in Cyrillic. Search works only by titles, but not by content.

**Reply by statistic1900** [reply]
Did a fresh install of the latest version and everything works correctly now

**Reply by Philipinho** [reply]
@masterbuck, @mstake01 can you please confirm from your browser's "WS" websocket network tab if the "/collab" path transmit data when you type? It could be that the editor changes do not get persisted to the database.

**Reply by mstake01** [reply]
@Philipinho On the advice above I did a clean install today and everything worked perfectly! Thank you for a amazing product!

**Comment by masterbuck** [comment, marked as answer]
Well, @Philipinho, the answer was somewhere in your question to check web sockets.

I went back to my reverse proxy (via Nginx Proxy Manager) and activated the web sockets, which made the search work... strange behavior! After editing all pages once (just add and remove a letter), the search worked on page contents as well as titles!

Problem solved for me!

**Comment by catch6** [comment]
I am using Chinese, and I can't find any result when I search. Do I need to make additional configurations for PostgreSQL? Are there any examples?

<img width="744" alt="image" src="https://github.com/user-attachments/assets/86fe1806-1578-43df-b462-a5535e62a73d" />
